winres/winres.json

```diff
@@ -0,0 +1,28 @@
+{
+  "RT_VERSION": {
+    "#1": {
+      "0000": {
+        "fixed": {
+          "file_version": "0.0.0.1",
```

Comment:
Do we need to update this to set the version?

Reply:
yes, it'd be nice to get rid of  the json file, will try out the suggestion from https://github.com/crc-org/win32-background-launcher/pull/1#discussion_r1377504359